tfhe/src/shortint/ciphertext/mod.rs

```diff
@@ -97,11 +97,11 @@ impl Degree {
             }
         }
 
-        Degree(result)
+        Self(result)
     }
 
     #[allow(dead_code)]
-    pub(crate) fn after_pbs<F>(&self, f: F) -> Degree
+    pub(crate) fn after_pbs<F>(&self, f: F) -> Self
```

Comment:
this can probably be removed as we have the degree in the accumulator directly

Reply:
in a follow up PR potentially

Reply:
https://github.com/zama-ai/tfhe-rs-internal/issues/316